### ISSUE TITLE
Modifier keys stay pushed while using a VM in the client.

### ISSUE THREAD

**Issue by Vrakfall** [symless/synergy#5281]
Hello,

Using a virtual machine on the client often becomes a nightmare when you have to press modifier keys like `Alt`. They often stay pressed (on the client-side) and it's really hard to make them be un-pressed because it's almost impossible to click anything on some windows managers while having the `Alt` key pressed (in mine, it's force the dragging of the window).

The virtual machine software is VMWare Workstation 12.1.0 (Linux version).

I'm using Windows 7 as Synergy server and Linux Mint 17.3 (Cinnamon window manager) as client. (Btw, I have some other bugs using this setup, I might do a ticket at some point. I juste don't have the time to reproduce it atm.)

The logs don't output anything particular.

How can I help you having more information about this bug?

**Comment by XinyuHou** [comment]
@Vrakfall 
Is this reliably reproducible?

**Comment by rougegoat** [comment]
I have this reliably happen on my server Windows 10 x64 + client OS X 10.11.4 setup when I lock the machines and come back.  In my situation, it's usually either Shift or Control that gets locked down even if nothing pressed it prior.  Stopping and restarting the service on either the client or server side usually fixes it, but that means I have to restart it every time I return to my machine.

**Comment by Vrakfall** [comment]
@XinyuHou Yes, this does happen very easily if done as describe. Only a few minutes are required to get annoyed by this bug.

**Comment by cjbats** [comment]
I am seeing this also. 

I am using win10 as the server and Mint 18.2 with cinnamon as the client. In Mint I am running vmware workstation 12.5 pro. If I hit ctrl + alt to get out of the console of a vm, the ctrl key is forever pushed down. I must restart both the server and client to reset.
